Tests/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ if (BUILD_WITH_FLAP)
     target_link_libraries(ExtDataDriver.x PRIVATE OpenMP::OpenMP_Fortran)
   endif ()
   set_target_properties(ExtDataDriver.x PROPERTIES Fortran_MODULE_DIRECTORY ${MODULE_DIRECTORY})
+  target_compile_definitions (ExtDataDriver.x PRIVATE $<$<BOOL:${USE_EXTDATA2G}>:BUILD_WITH_EXTDATA2G>)
```

Comment:
As per my earlier comment, my thought would be to build two executables here:  ExtDataDriver.x and ExtDataDriver2G.x.  Source code is identical but one "USE"s the old and one "USE"s the 2G via the compile time switch.

Reply:
added a runtime switch so I don't have to mess with two executables just like in capgridcomp